.github/workflows/ci.yml

```diff
@@ -41,10 +41,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.7
```

Comment:
Let's bump this to the new min version instead. If we want to test 3.9 as well, we can use a matrix strategy.